docs/installation/docker-available-configuration.md

```diff
@@ -0,0 +1,228 @@
+file that contains the changes you've made.
+
+### TAG
+
+The version of the `isle-buildkit` images to use. Non `isle-buildkit` images have their versions specified explicitly
+in their respective docker-compose files.
+
+By default, this is set to the latest stable release of `isle-buildkit`.
+
+If you change this setting, you need to `make -B docker-compose.yml` to generate a new `docker-compose.yml`
+file that contains the changes you've made.
+
+### USE_SECRETS
+
+Whether or not you want to use secrets.  For demo and development environments, secrets are
+not required.  They are essential if you are running a production environment, though.
+
+Secrets are contained in the `secrets` folder withiin your `isle-dc` installation.  Each file represents
+an individual secret, and its contents are the value you're trying to protect.
+
+Available values for this setting are
+- `true`
+- `false`
+
+If you change this setting, you need to `make -B docker-compose.yml` to generate a new `docker-compose.yml`
+file that contains the changes you've made.
+
+### DRUPAL_INSTALL_PROFILE
+
+Which install profile to use when making an initial installation.  Valid values for this setting are the machine
+names of any Drupal profile.
+
+By default, this is set to `standard`.
+
+### INSTALL_EXISTING_CONFIG
+
+Set this to `true` if you want to install an existing Drupal site whose configuration was exported with 
+`drush config:export` 
+
+Available values for this setting are
+- `true`
+- `false`
+
+By default, this is set to `false`.
+
+If you set this to `true`, be sure to set `DRUPAL_INSTALL_PROFILE` to `minimal`.
+
+### INCLUDE_WATCHTOWER_SERVICE
+
+Whether or not to include Watchtower as a service.  When developing `isle-buildkit`, this is extremely useful and
+will auto-deploy new containers as you make changes.  You should _not_ use watchtower on prodution environments, though.
+
+Available values for this setting are
+- `true`
+- `false`
+
+By default, this is set to `true`.
+
+If you change this setting, you need to `make -B docker-compose.yml` to generate a new `docker-compose.yml`
+file that contains the changes you've made.
+
+### INCLUDE_ETCD_SERVICE
+
+Whether or not to include `etcd` as a service.
+
+Available values for this setting are
+- `true`
+- `false`
+
+By default, this is set to `false`.
+
+If you change this setting, you need to `make -B docker-compose.yml` to generate a new `docker-compose.yml`
+file that contains the changes you've made.
+
+### INCLUDE_CODE_SERVER_SERVICE
+
+Whether or not to include the `coder` IDE as a service. If you're developing on Islandora, this can
+be pretty useful.
+
+Available values for this setting are
+- `true`
+- `false`
+
+By default, this is set to `false`.
+
+If you change this setting, you need to `make -B docker-compose.yml` to generate a new `docker-compose.yml`
+file that contains the changes you've made.
+
```

Comment:
Is running the code server OK on production? If it's OK, fine to leave as is, but if there is a security risk (or other risk running it on production, there should be note to that effect like there is for watchtower.

Reply:
Updated both that and the `etcd` service.